### ISSUE TITLE
[package] sdl_image/2.0.5: Does not install on Github Actions ubuntu-18.04 anymore

### ISSUE THREAD

**Issue by zaslavskyv** [conan-io/conan-center-index#10503]
### Package and environment details
  * Package Name/Version: **sdl_image/2.0.5**
  * Operating System+version: **Github Actions Ubuntu 18.04**
  * Compiler+version: **GCC 7**
  * Conan version: **conan 1.47.0**
  * Python version: **Python 2.7.17**

### Conan profile
```
[settings]
os=Linux
os_build=Linux
arch=x86_64
arch_build=x86_64
compiler=gcc
compiler.version=7
compiler.libcxx=libstdc++
build_type=Release
[options]
[conf]
[build_requires]
[env]
```

### Steps to reproduce

I have a sample [repository](https://github.com/zaslavskyv/sdl_image_ubuntu_runner) with two files:
##### conanfile.txt
``` ini
[requires]
sdl_image/2.0.5
```
##### .github/workflows/pipeline.yml
``` yml
name: pipeline
on: push
jobs:
  build:
    runs-on: ubuntu-18.04
    env:
      CONAN_V2_MODE: 1
      CONAN_SYSREQUIRES_MODE: enabled
    steps:
      - uses: actions/checkout@v2
      - uses: turtlebrowser/get-conan@main
      - run: conan install . -if build
```

Conan [does not find](https://github.com/zaslavskyv/sdl_image_ubuntu_runner/runs/6166979852?check_suite_focus=true) a suitable binary regardless of the `build_type={Debug|Release}` or `sdl_image:shared={True|False}` options. However, to my opinion there are configurations on the Conan Center which could be used, like [sdl_image/2.0.5:98f4cb6663dcc8cefa2fd18ab814fec2260085b8](https://conan.io/center/sdl_image?tab=dependencies&os=Linux)? The same setup used to work a couple of months ago.

### Logs
<details><summary>Click to expand log</summary>

```
env:
  CONAN_V2_MODE: 1
  CONAN_SYSREQUIRES_MODE: enabled

sdl_image/2.0.5: Not found in local cache, looking in remotes...
sdl_image/2.0.5: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
sdl_image/2.0.5: Downloaded recipe revision 0
sdl/2.0.16: Not found in local cache, looking in remotes...
sdl/2.0.16: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
sdl/2.0.16: Downloaded recipe revision 0
libiconv/1.16: Not found in local cache, looking in remotes...
libiconv/1.16: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libiconv/1.16: Downloaded recipe revision 0
libalsa/1.2.5.1: Not found in local cache, looking in remotes...
libalsa/1.2.5.1: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libalsa/1.2.5.1: Downloaded recipe revision 0
pulseaudio/14.2: Not found in local cache, looking in remotes...
pulseaudio/14.2: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
pulseaudio/14.2: Downloaded recipe revision 0
libsndfile/1.0.31: Not found in local cache, looking in remotes...
libsndfile/1.0.31: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libsndfile/1.0.31: Downloaded recipe revision 0
ogg/1.3.5: Not found in local cache, looking in remotes...
ogg/1.3.5: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
ogg/1.3.5: Downloaded recipe revision 0
vorbis/1.3.7: Not found in local cache, looking in remotes...
vorbis/1.3.7: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
vorbis/1.3.7: Downloaded recipe revision 0
flac/1.3.3: Not found in local cache, looking in remotes...
flac/1.3.3: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
flac/1.3.3: Downloaded recipe revision 0
opus/1.3.1: Not found in local cache, looking in remotes...
opus/1.3.1: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
opus/1.3.1: Downloaded recipe revision 0
libcap/2.62: Not found in local cache, looking in remotes...
libcap/2.62: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libcap/2.62: Downloaded recipe revision 0
libtool/2.4.6: Not found in local cache, looking in remotes...
libtool/2.4.6: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libtool/2.4.6: Downloaded recipe revision 0
automake/1.16.4: Not found in local cache, looking in remotes...
automake/1.16.4: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
automake/1.16.4: Downloaded recipe revision 0
autoconf/2.71: Not found in local cache, looking in remotes...
autoconf/2.71: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
autoconf/2.71: Downloaded recipe revision 0
m4/1.4.19: Not found in local cache, looking in remotes...
m4/1.4.19: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
m4/1.4.19: Downloaded recipe revision 0
xorg/system: Not found in local cache, looking in remotes...
xorg/system: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
xorg/system: Downloaded recipe revision 0
openssl/1.1.1n: Not found in local cache, looking in remotes...
openssl/1.1.1n: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
openssl/1.1.1n: Downloaded recipe revision 0
opengl/system: Not found in local cache, looking in remotes...
opengl/system: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
opengl/system: Downloaded recipe revision 0
nas/1.9.4: Not found in local cache, looking in remotes...
nas/1.9.4: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
nas/1.9.4: Downloaded recipe revision 0
wayland/1.20.0: Not found in local cache, looking in remotes...
wayland/1.20.0: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
wayland/1.20.0: Downloaded recipe revision 0
libffi/3.4.2: Not found in local cache, looking in remotes...
libffi/3.4.2: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libffi/3.4.2: Downloaded recipe revision 0
libxml2/2.9.13: Not found in local cache, looking in remotes...
libxml2/2.9.13: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libxml2/2.9.13: Downloaded recipe revision 0
zlib/1.2.11: Not found in local cache, looking in remotes...
zlib/1.2.11: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
zlib/1.2.11: Downloaded recipe revision 0
expat/2.4.8: Not found in local cache, looking in remotes...
expat/2.4.8: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
expat/2.4.8: Downloaded recipe revision 0
xkbcommon/1.4.0: Not found in local cache, looking in remotes...
xkbcommon/1.4.0: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
xkbcommon/1.4.0: Downloaded recipe revision 0
wayland-protocols/1.24: Not found in local cache, looking in remotes...
wayland-protocols/1.24: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
wayland-protocols/1.24: Downloaded recipe revision 0
egl/system: Not found in local cache, looking in remotes...
egl/system: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
egl/system: Downloaded recipe revision 0
libunwind/1.6.2: Not found in local cache, looking in remotes...
libunwind/1.6.2: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libunwind/1.6.2: Downloaded recipe revision 0
xz_utils/5.2.5: Not found in local cache, looking in remotes...
xz_utils/5.2.5: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
xz_utils/5.2.5: Downloaded recipe revision 0
libtiff/4.3.0: Not found in local cache, looking in remotes...
libtiff/4.3.0: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libtiff/4.3.0: Downloaded recipe revision 0
libdeflate/1.10: Not found in local cache, looking in remotes...
libdeflate/1.10: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libdeflate/1.10: Downloaded recipe revision 0
libjpeg/9d: Not found in local cache, looking in remotes...
libjpeg/9d: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libjpeg/9d: Downloaded recipe revision 0
jbig/20160605: Not found in local cache, looking in remotes...
jbig/20160605: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
jbig/20160605: Downloaded recipe revision 0
zstd/1.5.2: Not found in local cache, looking in remotes...
zstd/1.5.2: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
zstd/1.5.2: Downloaded recipe revision 0
libwebp/1.2.1: Not found in local cache, looking in remotes...
libwebp/1.2.1: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libwebp/1.2.1: Downloaded recipe revision 0
libpng/1.6.37: Not found in local cache, looking in remotes...
libpng/1.6.37: Trying with 'conancenter'...
Downloading conanmanifest.txt
Downloading conanfile.py
Downloading conan_export.tgz
libpng/1.6.37: Downloaded recipe revision 0
conanfile.txt: Installing package
Requirements
    autoconf/2.71 from 'conancenter' - Downloaded
    automake/1.16.4 from 'conancenter' - Downloaded
    egl/system from 'conancenter' - Downloaded
    expat/2.4.8 from 'conancenter' - Downloaded
    flac/1.3.3 from 'conancenter' - Downloaded
    jbig/20160605 from 'conancenter' - Downloaded
    libalsa/1.2.5.1 from 'conancenter' - Downloaded
    libcap/2.62 from 'conancenter' - Downloaded
    libdeflate/1.10 from 'conancenter' - Downloaded
    libffi/3.4.2 from 'conancenter' - Downloaded
    libiconv/1.16 from 'conancenter' - Downloaded
    libjpeg/9d from 'conancenter' - Downloaded
    libpng/1.6.37 from 'conancenter' - Downloaded
    libsndfile/1.0.31 from 'conancenter' - Downloaded
    libtiff/4.3.0 from 'conancenter' - Downloaded
    libtool/2.4.6 from 'conancenter' - Downloaded
    libunwind/1.6.2 from 'conancenter' - Downloaded
    libwebp/1.2.1 from 'conancenter' - Downloaded
    libxml2/2.9.13 from 'conancenter' - Downloaded
    m4/1.4.19 from 'conancenter' - Downloaded
    nas/1.9.4 from 'conancenter' - Downloaded
    ogg/1.3.5 from 'conancenter' - Downloaded
    opengl/system from 'conancenter' - Downloaded
    openssl/1.1.1n from 'conancenter' - Downloaded
    opus/1.3.1 from 'conancenter' - Downloaded
    pulseaudio/14.2 from 'conancenter' - Downloaded
    sdl/2.0.16 from 'conancenter' - Downloaded
    sdl_image/2.0.5 from 'conancenter' - Downloaded
    vorbis/1.3.7 from 'conancenter' - Downloaded
    wayland/1.20.0 from 'conancenter' - Downloaded
    wayland-protocols/1.24 from 'conancenter' - Downloaded
    xkbcommon/1.4.0 from 'conancenter' - Downloaded
    xorg/system from 'conancenter' - Downloaded
    xz_utils/5.2.5 from 'conancenter' - Downloaded
    zlib/1.2.11 from 'conancenter' - Downloaded
    zstd/1.5.2 from 'conancenter' - Downloaded
Packages
    autoconf/2.71:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Download
    automake/1.16.4:258f6f05ca54813b3d7180161753df7f9ccf7e1d - Download
    egl/system:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Download
    expat/2.4.8:d2aa7fde96817bd9d6d001c15e0479bf82133afe - Download
    flac/1.3.3:ef227c7ca8be49f078e2c7c4a6cebbac879b693a - Download
    jbig/20160605:d8fff784e71b40316e752103c83b4698d094b5e3 - Download
    libalsa/1.2.5.1:bff9a7d5ab267a535181ec5b36392417340ff035 - Download
    libcap/2.62:d50a0d523d98c15bb147b18fa7d203887c38be8b - Download
    libdeflate/1.10:d50a0d523d98c15bb147b18fa7d203887c38be8b - Download
    libffi/3.4.2:d50a0d523d98c15bb147b18fa7d203887c38be8b - Download
    libiconv/1.16:d50a0d523d98c15bb147b18fa7d203887c38be8b - Download
    libjpeg/9d:d50a0d523d98c15bb147b18fa7d203887c38be8b - Download
    libpng/1.6.37:9378b660d2155d53cea1126fa13164a3bfa59f2c - Download
    libsndfile/1.0.31:f4a930eea317b3d72a911436ea018737e1c780fe - Download
    libtiff/4.3.0:8311e1632d035c91234ae06c2588dd1f1f1da4e6 - Download
    libtool/2.4.6:6fbe64f35b9d3eb3f534cbf1646451f40f718a31 - Download
    libunwind/1.6.2:13388f4c6f64aa5c3223fdf4c310f7c2ba6e66c9 - Download
    libwebp/1.2.1:6322a9a56f029bb63526b01c9e32f1e334e3f0cb - Download
    libxml2/2.9.13:07753a3ecbad874f99f63d243fbb690a067a17be - Download
    m4/1.4.19:24647d9fe8ec489125dfbae4b3ebefaf7581674c - Download
    nas/1.9.4:a157a365c62ea5c47b862330c1c64b9785564910 - Download
    ogg/1.3.5:d50a0d523d98c15bb147b18fa7d203887c38be8b - Download
    opengl/system:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Download
    openssl/1.1.1n:d50a0d523d98c15bb147b18fa7d203887c38be8b - Download
    opus/1.3.1:d50a0d523d98c15bb147b18fa7d203887c38be8b - Download
    pulseaudio/14.2:39d1b4635fccdc89a4bd1ce0fde020eab20b0a67 - Download
    sdl/2.0.16:9a7cae42346dc6f07cac05c94d3547b4a2758040 - Download
    sdl_image/2.0.5:41a68900e7459cbb145d4fdc6f40cf115c4462b6 - Missing
    vorbis/1.3.7:8d8cebdeda6ff46ba515968aa8b099a24769ca16 - Download
    wayland/1.20.0:00d8f2e37c9112ad8417921edf0c46944c44bbab - Download
    wayland-protocols/1.24:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Download
    xkbcommon/1.4.0:f490dc036fc8115d98660a8f9e70c53ec22e7aea - Download
    xorg/system:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Download
    xz_utils/5.2.5:d50a0d523d98c15bb147b18fa7d203887c38be8b - Download
    zlib/1.2.11:d50a0d523d98c15bb147b18fa7d203887c38be8b - Download
    zstd/1.5.2:f96ae8d528b66eb76c2b5ecf40a4723eedaf4659 - Download

Installing (downloading, building) binaries...

WARN: Remotes registry file missing, creating default one in /home/runner/.conan/remotes.json
WARN: libxml2/2.9.13: requirement zlib/1.2.12 overridden by wayland/1.20.0 to zlib/1.2.11 
WARN: libunwind/1.6.2: requirement zlib/1.2.12 overridden by sdl/2.0.16 to zlib/1.2.11 
WARN: libtiff/4.3.0: requirement zlib/1.2.12 overridden by sdl_image/2.0.5 to zlib/1.2.11 
WARN: libtiff/4.3.0: requirement libwebp/1.2.2 overridden by sdl_image/2.0.5 to libwebp/1.2.1 
WARN: libpng/1.6.37: requirement zlib/1.2.12 overridden by sdl_image/2.0.5 to zlib/1.2.11 
ERROR: Missing binary: sdl_image/2.0.5:41a68900e7459cbb145d4fdc6f40cf115c4462b6
sdl_image/2.0.5: WARN: Can't find a 'sdl_image/2.0.5' package for the specified settings, options and dependencies:
- Settings: arch=x86_64, build_type=Release, compiler=gcc, compiler.version=7, os=Linux
- Options: bmp=True, fPIC=True, gif=True, lbm=True, pcx=True, pnm=True, shared=False, svg=True, tga=True, with_libjpeg=True, with_libpng=True, with_libtiff=True, with_libwebp=True, xcf=True, xpm=True, xv=True, expat:char_type=char, expat:fPIC=True, expat:shared=False, flac:fPIC=True, flac:shared=False, jbig:build_executables=True, jbig:fPIC=True, jbig:shared=False, libalsa:disable_python=True, libalsa:fPIC=True, libalsa:shared=False, libcap:fPIC=True, libcap:psx_syscals=False, libcap:shared=False, libdeflate:fPIC=True, libdeflate:shared=False, libffi:fPIC=True, libffi:shared=False, libiconv:fPIC=True, libiconv:shared=False, libjpeg:fPIC=True, libjpeg:shared=False, libpng:api_prefix=, libpng:fPIC=True, libpng:shared=False, libpng:sse=True, libsndfile:experimental=False, libsndfile:fPIC=True, libsndfile:programs=True, libsndfile:shared=False, libsndfile:with_alsa=False, libsndfile:with_external_libs=True, libtiff:cxx=True, libtiff:fPIC=True, libtiff:jbig=True, libtiff:jpeg=libjpeg, libtiff:libdeflate=True, libtiff:lzma=True, libtiff:shared=False, libtiff:webp=True, libtiff:zlib=True, libtiff:zstd=True, libtool:fPIC=True, libtool:shared=False, libunwind:coredump=True, libunwind:fPIC=True, libunwind:minidebuginfo=True, libunwind:ptrace=True, libunwind:setjmp=True, libunwind:shared=False, libunwind:zlibdebuginfo=True, libwebp:fPIC=True, libwebp:near_lossless=True, libwebp:shared=False, libwebp:swap_16bit_csp=False, libwebp:with_simd=True, libxml2:c14n=True, libxml2:catalog=True, libxml2:docbook=True, libxml2:fPIC=True, libxml2:ftp=True, libxml2:html=True, libxml2:http=True, libxml2:iconv=True, libxml2:icu=False, libxml2:include_utils=True, libxml2:iso8859x=True, libxml2:legacy=True, libxml2:lzma=False, libxml2:mem-debug=False, libxml2:output=True, libxml2:pattern=True, libxml2:push=True, libxml2:python=False, libxml2:reader=True, libxml2:regexps=True, libxml2:run-debug=False, libxml2:sax1=True, libxml2:schemas=True, libxml2:schematron=True, libxml2:shared=False, libxml2:threads=True, libxml2:tree=True, libxml2:valid=True, libxml2:writer=True, libxml2:xinclude=True, libxml2:xpath=True, libxml2:xptr=True, libxml2:zlib=True, nas:fPIC=True, nas:shared=False, ogg:fPIC=True, ogg:shared=False, openssl:386=False, openssl:enable_weak_ssl_ciphers=False, openssl:fPIC=True, openssl:no_aria=False, openssl:no_asm=False, openssl:no_async=False, openssl:no_bf=False, openssl:no_blake2=False, openssl:no_camellia=False, openssl:no_cast=False, openssl:no_chacha=False, openssl:no_cms=False, openssl:no_comp=False, openssl:no_ct=False, openssl:no_deprecated=False, openssl:no_des=False, openssl:no_dgram=False, openssl:no_dh=False, openssl:no_dsa=False, openssl:no_dso=False, openssl:no_ec=False, openssl:no_ecdh=False, openssl:no_ecdsa=False, openssl:no_engine=False, openssl:no_filenames=False, openssl:no_gost=False, openssl:no_hmac=False, openssl:no_idea=False, openssl:no_md4=False, openssl:no_md5=False, openssl:no_mdc2=False, openssl:no_ocsp=False, openssl:no_pinshared=False, openssl:no_rc2=False, openssl:no_rfc3779=False, openssl:no_rmd160=False, openssl:no_rsa=False, openssl:no_seed=False, openssl:no_sha=False, openssl:no_sm2=False, openssl:no_sm3=False, openssl:no_sm4=False, openssl:no_sock=False, openssl:no_srp=False, openssl:no_srtp=False, openssl:no_sse2=False, openssl:no_ssl=False, openssl:no_ssl3=False, openssl:no_stdio=False, openssl:no_tests=False, openssl:no_threads=False, openssl:no_tls1=False, openssl:no_ts=False, openssl:no_whirlpool=False, openssl:openssldir=None, openssl:shared=False, opus:fPIC=True, opus:fixed_point=False, opus:shared=False, pulseaudio:fPIC=True, pulseaudio:shared=False, pulseaudio:with_alsa=True, pulseaudio:with_dbus=False, pulseaudio:with_glib=False, pulseaudio:with_openssl=True, pulseaudio:with_x11=True, sdl:alsa=True, sdl:arts=False, sdl:directfb=False, sdl:esd=False, sdl:fPIC=True, sdl:iconv=True, sdl:jack=False, sdl:libunwind=True, sdl:nas=True, sdl:opengl=True, sdl:opengles=True, sdl:pulse=True, sdl:sdl2main=True, sdl:shared=False, sdl:sndio=False, sdl:video_rpi=False, sdl:vulkan=True, sdl:wayland=True, sdl:x11=True, sdl:xcursor=True, sdl:xinerama=True, sdl:xinput=True, sdl:xrandr=True, sdl:xscrnsaver=True, sdl:xshape=True, sdl:xvm=True, vorbis:fPIC=True, vorbis:shared=False, wayland:enable_dtd_validation=True, wayland:enable_libraries=True, wayland:fPIC=True, wayland:shared=False, xkbcommon:fPIC=True, xkbcommon:shared=False, xkbcommon:with_wayland=True, xkbcommon:with_x11=True, xkbcommon:xkbregistry=True, xz_utils:fPIC=True, xz_utils:shared=False, zlib:fPIC=True, zlib:shared=False, zstd:fPIC=True, zstd:shared=False, zstd:threading=True
- Dependencies: sdl/2.0.16, libtiff/4.3.0, libjpeg/9d, libpng/1.6.37, libwebp/1.2.1, zlib/1.2.11
- Requirements: libjpeg/9d, libpng/1.Y.Z, libtiff/4.Y.Z, libwebp/1.Y.Z, sdl/2.Y.Z, zlib/1.Y.Z
- Package ID: 41a68900e7459cbb145d4fdc6f40cf115c4462b6

ERROR: Missing prebuilt package for 'sdl_image/2.0.5'
Use 'conan search sdl_image/2.0.5 --table=table.html -r=remote' and open the table.html file to see available packages
Or try to build locally from sources with '--build=sdl_image'

More Info at 'https://docs.conan.io/en/latest/faq/troubleshooting.html#error-missing-prebuilt-package'
```

</details>

**Comment by ericLemanissier** [comment]
This should have been fixed with https://github.com/conan-io/conan-center-index/pull/10461

**Comment by zaslavskyv** [comment]
> This should have been fixed with #10461

Yes, just ran [the build](https://github.com/zaslavskyv/sdl_image_ubuntu_runner/runs/6211334280?check_suite_focus=true) successfully. Thank you guys!
